src/store/slices/users/fetchUsers.ts

```diff
@@ -19,3 +31,24 @@ export const fetchData = createAsyncThunk(
     return data;
   }
 );
+
+export const fetchSignedInUSer = createAsyncThunk(
+  'data/fetchSignedInUSer',
+  async (): Promise<Partial<GlobalState>> => {
+    const data: any = [];
+    try {
+      onAuthStateChanged(auth, (user) => {
+        if (user) {
+          data.push(user);
+          console.log('User is signed in');
```

Comment:
Nu lasa console.log-uri aiurea nu vrem sa le bagam in productie. Trebuie sa fii mai atent decat de obicei